### ISSUE TITLE
CEP Extension does not move away from start page in newest Photoshop update 21.1.1

### ISSUE THREAD

**Issue by rh78** [Adobe-CEP/CEP-Resources#282]
Hello,

we noticed strange behavior in newest version of Photoshop. Please note that this behavior does not apply to e.g. Photoshop CC 2019, and as far as we see, it also does not apply to Illustrator / InDesign etc.

Behavior is as follows: we launch the plugin in Photoshop (to exclude side effects, we just have an ultra-simple panel.html, containing only a link). Usually if you click the link, the page moves - BUT now the plugin just immediately loads again the panel.html. There is no way to move away from the page (e.g. form POSTs do nothing but just reload the page, links do nothing but just reload the page, ...)

This is the CEP HTML Engine log (level 6) - clicks marked with ==> CLICK
[CEPHtmlEngine9-PHXS-21.1.1-io.smint.clc.adobe.extension.log](https://github.com/Adobe-CEP/CEP-Resources/files/4389481/CEPHtmlEngine9-PHXS-21.1.1-io.smint.clc.adobe.extension.log)

This is the CEP HTML Engine renderer log
[CEPHtmlEngine9-PHXS-21.1.1-io.smint.clc.adobe.extension-renderer.log](https://github.com/Adobe-CEP/CEP-Resources/files/4389483/CEPHtmlEngine9-PHXS-21.1.1-io.smint.clc.adobe.extension-renderer.log)

This is the CEP9-PHXS.log file - clicks marked with ==> CLICK
[CEP9-PHXS.log](https://github.com/Adobe-CEP/CEP-Resources/files/4389485/CEP9-PHXS.log)

This is a screenshot of the traffic in DevTools:
![devtools_traffic](https://user-images.githubusercontent.com/16898150/77688216-fb9bdb00-6f9f-11ea-8465-f51df7dd2cf0.png)

This is the panel and the code of the panel.html file:
![panel_code](https://user-images.githubusercontent.com/16898150/77688344-3998ff00-6fa0-11ea-8359-7ee26aee2fab.png)

Please advise!

**Comment by Fidomaster01** [comment]
Hello,

My problem is similar, but in after effects.
I have experienced, if i onclick a button that it shows to an external link, everything will be ok.
My environment is: 
Windows 10 Enterprise - 1909
After Effects - 17.0.6

Thnx for advise!

**Comment by kareemmyk** [comment]
@rh78 ... what is the environment you are running this with?
Mac version or windows version. I see @Fidomaster01 is reporting after effects on win 10

**Comment by kareemmyk** [comment]
@rh78 ... are you expecting the url to be launched on a browser instead of the CEP extension?

I was able to launch both the localhost and external url within the Panel.

**Comment by Fidomaster01** [comment]
@rh78  On my panel are few buttons. All buttons url are define localhost, and one button define external url (<button  onclick="window.cep.util.openURLInDefaultBrowser('http://gfx/blablabla ). 
When i push either button, its only a white flash and stay the home page, but i push the button with external link, everything will be ok, all localhost button will be work well, and all localhost page open.
I read lot of forums, but i didn't  found nothing. Solution: i created my panel with iframe, and everything work well. All buttons with localhost url work great for the first time.
I hope it will help you, but i still don't understand what causes the error.

**Comment by rh78** [comment]
We worked around the problem by using an IFrame - within the Iframe we have no issues. But it is complex of course.

**Comment by Fidomaster01** [comment]
Exactly. but I also want to know what has changed. One version later, everything is still good

**Comment by znap026** [comment]
I think I'm having the same issue. It works fine on mac but crashes and reloads on windows. I really don't want to have to rewrite to use an iFrame. 

[https://community.adobe.com/t5/premiere-pro/cep-plugin-development-window-top-location-replace-crashes-cephtmlengine-exe/td-p/11154169?page=1](https://community.adobe.com/t5/premiere-pro/cep-plugin-development-window-top-location-replace-crashes-cephtmlengine-exe/td-p/11154169?page=1)
